api/src/main/resources/workbench-api.yaml

```diff
@@ -2341,25 +2341,58 @@ paths:
           description: The updated Institution.
           schema:
             "$ref": "#/definitions/Institution"
-  "/v1/institutions/{shortName}/checkEmail/{email}/":
+  "/v1/institutions/{shortName}/checkEmail":
     parameters:
     - in: path
       name: shortName
       type: string
       description: 'The short name / key of the institution to check, e.g. "VUMC"'
       required: true
-    - in: path
-      name: email
-      type: string
-      description: Institutional contact email address to check.
-      required: true
-    get:
+    post:
       tags:
         - institution
       operationId: checkEmail
       description: Checks whether the given email address is a verified member of an institution.
       x-aou-note: This endpoint is publicly-accessible and called by clients during user registration.
       security: []
+      parameters:
+      - in: body
+        name: contactEmail
+        description: Institutional contact email address to check.
+        required: true
+        schema:
+          type: object
+          required:
+            - value
+          properties:
+            value:
+              type: string
+              description: Institutional contact email address to check.
+
+      responses:
+        200:
+          description: Whether the email is a valid member of the institution.
+          schema:
+            "$ref": "#/definitions/CheckEmailResponse"
+  "/v1/institutions/{shortName}/checkEmail/{email}/":
```

Comment:
keeping the old endpoint so as not to immediately break anything reliant on it.  https://precisionmedicineinitiative.atlassian.net/browse/RW-4948 removes it.

Reply:
Nice approach! I appreciate the attention to detail (and I'm sure Alex does too).